cli/src/feature.rs

```diff
@@ -957,6 +957,6 @@ fn process_activate(
         FEATURE_NAMES.get(&feature_id).unwrap(),
         feature_id
     );
-    rpc_client.send_and_confirm_transaction_with_spinner(&transaction)?;
-    Ok("".to_string())
+    let signature = rpc_client.send_and_confirm_transaction_with_spinner(&transaction)?;
+    Ok(format!("Signature: {signature}"))
```

Comment:
There are actually a couple options here that would be more consistent with other parts of solana-cli, and automatically support `--output json`:

```suggestion
    let signature = rpc_client.send_and_confirm_transaction_with_spinner(&transaction)?;
    let signature = CliSignature {
        signature: signature.to_string(),
    };
    Ok(config.output_format.formatted_string(&signature))
```

This is also an option, and is cool because it adds nice parsing for instruction errors, but will need to be rewritten in the near future when we start using a bpf program for feature gate activations:
```suggestion
    let result = rpc_client.send_and_confirm_transaction_with_spinner(&transaction);
    log_instruction_custom_error::<SystemError>(result, config)
```

(PS. I've never tried putting two suggestions in one comment before. Hopefully it Just Works ™️ )